### ISSUE TITLE
Manual connector delete

### ISSUE THREAD

**Issue by ko1N** [memflow/memflowup#7]
If connectors are deleted by the user theyll still be seen as "installed". The filename should be checked in this case and they should be removed from the registry.

**Comment by h33p** [comment]
I disagree with the exact solution. I wouldn't automatically clean the registry just because one file is missing. I would mark the connector as `installed (broken)` or `installed (externally deleted)` that the users could then run `uninstall` or `fix` command to reconcile the state.

**Comment by ko1N** [comment]
This is now solved in memflowup 0.2.0 - If a connector is installed manually without an appropriate .meta file it will be caught by `memflowup plugins clean` and deleted. If the user wants to install the plugin properly from source we can do:

```
memflowup build https://github.com/memflow/memflow-coredump
```
to download and install from a git repository (even non official ones), or
```
memflowup build --path .
```
to properly install a connector from a locally checked out repo.
